### DISCUSSION TITLE
Cannot add existing collaborator

### DISCUSSION THREAD

**Discussion by ironfrown** [community/community#124849] [Repositories] [answered]
### Select Topic Area

Question

### Body

I have just created a new public repo and uploaded files into it, so it works. Now I am trying to add a colleague, who is my collaborator in another repo. When I add his ID in the search field, he is found and I select him. His 'user/number' replaces his ID and when I select it, I get "Could not find a GitHub account matching user/number". What is going on?

**Comment by ironfrown** [comment, marked as answer]
It seems, this problem only exists on Google Chrome (Ubuntu 22.04). When I switched to Firefox, I had no problems!

**Reply by queenofcorgis** [reply]
Thanks for updating with a workaround!
